### PR TITLE
Added ElevatedButton code sample

### LINKED ISSUE #75385: ElevatedButton needs dartpad demo on official docs
## Use case
ElevatedButton documentation doesn't have dartpad demo though RaisedButton had it.
Demo will be helpful for new devs to understand the widget.

## Proposal
I'm thinking to adding dartpad demo like [RaisedButton](https://api.flutter.dev/flutter/material/RaisedButton-class.html).
But, where should I put demo codes?

### LINKED ISSUE #75642: TextButton, ElevatedButton, and OutlinedButton need API doc samples
Although many useful examples can be found in flutter.dev/go/material-button-migration-guide and in https://github.com/flutter/flutter/pull/59702 as well as the [ButtonStyle API doc](https://api.flutter.dev/flutter/material/ButtonStyle-class.html), the API docs for the new Button classes need examples as well.

### PR DESCRIPTION
**Pull request by Abhishek01039**
related to https://github.com/flutter/flutter/issues/75642
fixes https://github.com/flutter/flutter/issues/75385
fixes https://github.com/flutter/flutter/issues/75642

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter/lib/src/material/elevated_button.dart [hunk 1]
```diff
@@ -47,6 +47,40 @@ import 'theme_data.dart';
...
+/// ElevatedButton.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return Center(
+///     child: Column(
+///       mainAxisSize: MainAxisSize.min,
+///       children: <Widget>[
+///         ElevatedButton(
+///            style: ElevatedButton.styleFrom(
```
Comment (HansMuller) [inline packages/flutter/lib/src/material/elevated_button.dart:62]: Might as well factor out the style parameter as a final variable.

File: packages/flutter/lib/src/material/elevated_button.dart [hunk 2]
```diff
@@ -47,6 +47,40 @@ import 'theme_data.dart';
 /// If [onPressed] and [onLongPress] callbacks are null, then the
 /// button will be disabled.
 ///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows how to render a disabled ElevatedButton, an enabled
```
Comment (HansMuller) [inline packages/flutter/lib/src/material/elevated_button.dart:52]: This sample produces an enabled and a disabled ElevatedButton.

Comment (Abhishek01039) [reply packages/flutter/lib/src/material/elevated_button.dart]: Thanks @HansMuller 
can you please review it now?
Thanks

Comment (HansMuller) [reply packages/flutter/lib/src/material/elevated_button.dart]: The style variable should be defined in the build method and it looks like you need to clean up some trailing white space.

**Review by HansMuller — APPROVED** [review]
LGTM

**Pull request merged**